pkg/system/phase2_creating.go

```diff
@@ -511,6 +511,16 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &configMapVolumeMounts)
 			}
+
+			// We are mounting the log path inside 'noobaa-log-processor' container here
+			// TODO: need to update the logic and actual params here, currently hardcoded
+			if false {
+				bucketLogVolumeMounts := []corev1.VolumeMount{{
+					Name:      "aayush-pvc",
```

Comment:
let's give this PVC a better name -  bucket-logging-volume, or something similar...

Reply:
sure, Thanks.